### PR TITLE
Preparations for RC1 release

### PR DESCRIPTION
**Pull request by slahirucd7**
### Purpose
$subject

### Issues
N/A
Fixes #

### Automation tests
 - Unit tests added: No
 - Integration tests added: No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

File: resources/docker-compose/apim/docker-compose.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 version: "3.7"
 services:
   apim:
-    image: wso2am/wso2am:4.1.0-rc2-alpine
```
Comment (slahirucd7) [inline resources/docker-compose/apim/docker-compose.yaml:4]: Please check the image name in here.

File: resources/license/LICENSE-ENFORCER.txt [hunk 2]
```diff
@@ -15,7 +15,7 @@ below is and where the actual text of the license can be found.
 Name                                                                                                Type           License
 _________________________________________________________________________________________________________________________________________________
 opentelemetry-extension-trace-propagators-1.6.0.jar                                                 jar            apache2
-org.wso2.choreo.connect.enforcer.commons-1.1.0-beta2.jar                                            bundle         apache2
+org.wso2.choreo.connect.enforcer.commons-1.1.0.jar                                            bundle         apache2
```
Comment (slahirucd7) [inline resources/license/LICENSE-ENFORCER.txt:18]: Please check this license text change.

File: integration/test-integration/src/test/resources/dockerCompose/backend-service-tls-with-opa-server.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
       - "5006:5006"
   mockBackend:
-    image: wso2/choreo-connect-mock-backend:1.1.0-beta2-SNAPSHOT
```
Comment (slahirucd7) [inline integration/test-integration/src/test/resources/dockerCompose/backend-service-tls-with-opa-server.yaml:3]: Please check this image name change.

File: release-note.html [hunk 4]
```diff
@@ -23,42 +23,45 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 <head>
     <meta content="text/html; charset=iso-8859-1" http-equiv="content-type"/>
-    <title>Choreo Connect 1.0.0 - 5th January 2022</title>
+    <title>Choreo Connect 1.1.0 - 5th January 2022</title>
```
Comment (pubudu538) [inline release-note.html:26]: Date?

File: resources/docker-compose/apim/docker-compose.yaml [hunk 5]
```diff
@@ -1,7 +1,7 @@
...
+    image: wso2/wso2am:4.1.0alpine
```
Comment (pubudu538) [inline resources/docker-compose/apim/docker-compose.yaml:4]: Images - 

wso2/wso2am:4.1.0-alpine
wso2am/wso2am:4.1.0-alpine

File: integration/test-integration/src/test/resources/dockerCompose/backend-service-tls-with-opa-server.yaml (hunk 3 above)
Comment (renuka-fernando) [reply integration/test-integration/src/test/resources/dockerCompose/backend-service-tls-with-opa-server.yaml:3]: yes correct

File: release-note.html [hunk 6]
```diff
@@ -23,42 +23,45 @@
...
-            <li>Custom filters</li>
-            <li>Tracing with open tracing</li>
-            <li>Immutable gateway support</li>
+            <li>Mock API implementation support considering API definition.</li>
+            <li>Operational policies support with WSO2 API-M.</li>
+            <li>Request payload passing from router to enforcer.</li>
+            <li>Git integration support.</li>
+            <li>Open Policy Agent (OPA) validation support.</li>
+            <li>Support for default versioned APIs.</li>
+            <li>Analytics publishing for WebSocket APIs.</li>
```
Comment (renuka-fernando) [inline release-note.html:45]: @pubudu538 plz check
```suggestion
            <li>Mock API implementation support considering API definition.</li>
            <li>API Policy support with WSO2 API-M.</li>
            <li>Request payload passing from router to enforcer.</li>
            <li>Git integration support.</li>
            <li>Open Policy Agent (OPA) validation.</li>
            <li>Default versioned APIs.</li>
            <li>Analytics publishing and rate limitting for WebSocket APIs.</li>
```

Comment (pubudu538) [reply release-note.html:45]: +1

File: release-note.html [hunk 7]
```diff
@@ -23,42 +23,45 @@
...
-            <li>Per endpoint certificate support</li>
-            <li>Backend JWT support with internal keys</li>
-            <li>Support subscription validation with API keys</li>
-            <li>Support endpoint referencing</li>
+            <li>API policy version support.</li>
+            <li>Improved WebSocket API handling.</li>
+            <li>Configurable DNS refresh rate.</li>
+            <li>Log correlation in enforcer.</li>
+            <li>Operation level interceptor support.</li>
+            <li>Request payload passing to custom filters.</li>
```
Comment (pubudu538) [inline release-note.html:57]: We can remove request payload thing from this list as we already mentioned this in the feature list.

**Review by pubudu538 — APPROVED** [review]

**Pull request merged**